src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1053,5 +1058,41 @@ private void CompleteDatabaseUnloading(DocumentDatabase database)
 
             database.DatabaseShutdownCompleted.Set();
         }
+
+        private void CheckDatabasePathsIntersection(string databaseName, RavenConfiguration configuration)
+        {
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var allDatabasesRecords = _serverStore.Cluster.GetAllDatabases(context);
+                var parentDir = configuration.Core.DataDirectory.FullPath;
+
+                foreach (var currRecord in allDatabasesRecords)
+                {
+                    if (currRecord.DatabaseName.Equals(databaseName))
+                        continue;
+
+                    RavenConfiguration currConfiguration;
+                    try
+                    {
+                        currConfiguration = CreateDatabaseConfiguration(currRecord.DatabaseName, ignoreDisabledDatabase: true, 
+                            ignoreBeenDeleted: true, ignoreNotRelevant: true, databaseRecord: currRecord);
+                    }
+                    catch (Exception e)
+                    {
+                        currConfiguration = null;
+                        if (_logger.IsInfoEnabled)
+                            _logger.Info("Could not create database configuration", e);
+                    }
+
+                    var currDir = currConfiguration?.Core.DataDirectory.FullPath;
```

Comment:
Also check the TempPath and JournalsPath

Reply:
Done